downstream/modules/platform/con-gw-pluggable-authentication.adoc

```diff
@@ -4,14 +4,14 @@
 
 = Pluggable authentication
 
-Authentication is the process of verifying a user's identity to Ansible Automation Platform, which means establishing that a user is who they say they are. This can be done in a number of ways but would traditionally be associated with a username and password.
+Authentication is the process of verifying a user's identity to the {PlatformNameShort} (that is, to establish that a user is who they say they are). This can be done in a number of ways but would traditionally be associated with a `username` and `password`.
 
-In Ansible Automation Platform 2.4, you could configure external authentication sources such as LDAP or SAML, in automation controller. Ansible Automation Platform 2.5 uses a pluggable authentication system with a configuration wizard that provides a common, simplified method of configuring different types of authenticators, such as LDAP and SAML. The pluggable system also allows you to  configure multiple authenticators of the same type.  
+In {PlatformNameShort} 2.4, you could configure external authentication sources such as LDAP or SAML, in {ControllerName}. {PlatformNameShort} {PlatformVers} uses a pluggable authentication system with a configuration wizard that provides a common, simplified method of configuring different types of authenticators, such as LDAP and SAML. The pluggable system also allows you to configure multiple authenticators of the same type.  
 
-In the pluggable system we have three concepts:
+In the pluggable system we have a couple of concepts:
 
-Authenticator plugin:: A plugin allows Ansible Automation Platform to connect to a source system, such as LDAP or SAML. Ansible Automation Platform is shipped with a variety of authenticator plugins and more can be added. Authenticator plugins are similar to Ansible collections, in that all of the required code is in a package and can be versioned independently if needed. You can write and add new custom plugins to the system
+Authenticator Plugin:: A plugin allows {PlatformNameShort} to connect to a source system, such as, LDAP or SAML. {PlatformNameShort} includes a variety of authenticator plugins and more can be added. Authenticator plugins are similar to Ansible collections, in that, all of the required code is in a package and can be versioned independently if needed. 
```

Comment:
Would remove commas after "such as" and "in that"

Reply:
Fixed.